allensdk/brain_observatory/behavior/trials_processing.py

```diff
@@ -628,7 +627,7 @@ def get_time(exp_data):
 def data_to_licks(data, time):
     lick_frames = data['items']['behavior']['lick_sensors'][0]['lick_events']
     lick_times = time[lick_frames]
-    return pd.DataFrame(data={"frame": lick_frames, 'time': lick_times})
+    return pd.DataFrame(data={"frame": lick_frames, 'timestamps': lick_times})
```

Comment:
Small nit:

Can we make the order `timestamps` then `frame` so that it's consistent with how it's return by `get_licks` in `BehaviorDataTransforms`?

Reply:
done